articles/quickstart/native/ios-objc/05-authorization.md

```diff
@@ -0,0 +1,59 @@
+---
```

Comment:
Overall for this article, I think the way to do it now is adding custom claims to the access_token. i.e. `https://myapp.com/users/role=admin`. I'll review this with you the next week.